src/daemon/utils_cache.c

```diff
@@ -641,7 +644,7 @@ int uc_get_value_by_name(const char *name, value_t *ret_values) {
     if (ce->state == STATE_MISSING) {
       status = -1;
     } else {
-      *ret_values = ce->values_raw;
+      *ret_values = typed_value_clone(ce->values_raw).value;
```

Comment:
At your option, a `value_t value_clone(value_t v)` might be easier to read.

Reply:
Unfortunately, we can't do it like this as then we won't pass any information about type to clone function. What we can do is to create a function `value_t value_clone(typed_value_t)`. Do you think it'd be better?